Create phd-course-dynamics-complex-systems.md (#105)
dai-dinamici/dai-dinamici.github.io: content/posts/phd-course-dynamics-complex-systems.md

+++ 
title = "Ph.D. course 'Dynamics of complex systems'" 
date = "2024-04-05 11:00:04" 
categories = ["news", "events"] 
+++

The Ph.D. School in Mathematics of the *University of Pisa* offers the course on the **'Dynamics of complex systems'** by Stefano Galatolo, Roberto Castorrini, Matteo Tanzi, and Bastien Fernandez. 

> The course is in presence but can also be followed remotely.

Author: Change into "The course is in person but..."

Reviewer: Why not
```suggestion
The course will be offered in an hybrid format.
```

Reviewer: ```suggestion
This course will be conducted in person, with the option for remote participation.
```
